src/content/drafts/code-review-nit-to-ecosystem-improvements.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "Code review nit to ecosystem improvements"
+teaser: "Reflecting on the power of lint rules"
+authors:
+  - captbaritone
+categories:
+  - Case Studies
+tags:
+  - Guest Post
+  - Rules
+---
+Four years ago, doing code review at work, I was surprised that [Flow](https://flow.org/) had not warned about a null check that had become unnecessary. This month [TypeScript 5.6](https://devblogs.microsoft.com/typescript/announcing-typescript-5-6/#disallowed-nullish-and-truthy-checks1) released with validation rules that disallows nullish and truthy checks which uncovered nearly 100 existing bugs in the top 800 TypeScript repos on GitHub.
+
+The two events are connected, because that moment in code review four years ago lead me to write [a lint rule](https://eslint.org/blog/2022/07/interesting-bugs-caught-by-no-constant-binary-expression/) which was part of the inspiration for the TypeScript features.
```

Comment:
```suggestion
The two events are connected, because that moment in code review four years ago led me to write [a lint rule](https://eslint.org/blog/2022/07/interesting-bugs-caught-by-no-constant-binary-expression/) which was part of the inspiration for the TypeScript features.
```